_maps/pubbystation.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
`build_pipeline(): /obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden/layer4 added to a pipenet while still having one. (pipes leading to the same spot stacking in one turf) around Medbay Lobby.`
<hr>
There's a lot of these errors coming up, but as far as I can tell they're all being caused by this one tile in the medbay lobby:

![image](https://user-images.githubusercontent.com/57483089/169621084-eeb234d6-97e9-4c5c-907d-e74da6c8faeb.png)
Deleting one of the duplicate supply pipes from that tile seems to fix it.